wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclRecordReader.java

```diff
@@ -442,7 +442,10 @@ public void enterComment(final EclRecordParser.CommentContext ctx) {
     @Override
     public void exitAnnotation(final EclRecordParser.AnnotationContext ctx) {
         super.exitAnnotation(ctx);
-        currentrec.getAnnotations().add(new DFUDataColumnAnnotation(annotationName, annotationParams));
+        if (currentrec.getChildColumns().size() > 0) {
+            final DFUDataColumnInfo info = currentrec.getChildColumns().get(currentrec.getChildColumns().size() - 1);
```

Comment:
I don't change the reference, so IMHO, yes.  If you look at some of my code, I always use final unless I'm going to change a reference (which I try to avoid doing).